util/helpfulIO.py

```diff
@@ -23,11 +29,22 @@ def __init__(
     ) -> None:  # depending on if its in simulation or the real motor, different motors will be used
         self.name = name
         self.motor = (
-            WPI_TalonFX(realId, canbus) if RobotBase.isReal() else PWMVictorSPX(simId)
+            WPI_TalonFX(realId, canbus) if RobotBase.isReal() else WPI_TalonFX(realId)
```

Comment:
I'm assuming this stuff will eventually disappear anyway but looks good for now

Reply:
should be replaced without the if statement

Reply:
Not sure how the simulated Falcon handles the canbus argument. Largely irrelevant once https://github.com/1757WestwoodRobotics/2022-RapidReact/tree/system-performance-optimizations is brought in (after being brought up to date).